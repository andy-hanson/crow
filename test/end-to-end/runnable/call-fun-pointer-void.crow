import
	crow/io/print: out, print

main void() summon
	xs string mut[] = ()
	res = xs.get-fun[xs ~= "arg"]
	out print (" " join xs.move-to)
	out print "{res}"

get-fun nat function()(xs string mut[])
	xs ~= "get-fun"
	&foo

foo nat()
	3
