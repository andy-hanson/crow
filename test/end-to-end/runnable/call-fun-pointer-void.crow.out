get-fun arg
3
